### ISSUE TITLE
Glowing Mushrooms Question

### ISSUE THREAD

**Issue by Teslachu** [Sunekaer/stoneBlock#760]
I saw you previously had disabled glowing mushrooms from world spawn... Then you later re-enabled them.  I am currently running StoneBlock version 1.0.31 and I checked the config - I can see they aren't disabled. I looked around in a ton of caves, ravines, and mineshafts (in the mining world), since I know things that would spawn in the overworld, often spawn there. However, I have yet to find any. Are these actually disabled or something? The world was generated on 1.3.1

**Comment by Teslachu** [comment]
Just an update- I was finally able make the RFTools Dimension setup, to build dimensions and such. I did find the glowing mushroom in an RFTools Dimension. I know one person on the server also found it that way. I am not sure if it should spawn in the mining dimension at all, but I know it is rare, even in the RFTools Dimension. I figured I'd update this, so you know that they can be found... Just not sure how early you want people to be able to find them and such or if they should be spawning in the mining world.
